BUILD

```diff
@@ -12,6 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@rules_license//rules:license.bzl", "license")
+
+package(
+    default_applicable_licenses = [":license"],
+    default_visibility = ["//visibility:public"],
+)
+
+license(
+    name = "license",
+    license_kinds = [
+        "@rules_license//licenses/spdx:Apache-2.0"
+    ],
+    license_text = "LICENSE",
```

Comment:
IMO this attribute should read `license_text_file`.  As written, a user might intuitively think that the license text is literally "LICENSE".

Reply:
I agree on this point. The attribute would be more explicit as `license_text_file`. I noticed that was the original name as described in the [license checking design doc](https://docs.google.com/document/d/1uwBuhAoBNrw8tmFs-NxlssI6VRolidGYdYqagLqHWt8/edit#heading=h.cgrjvasquhse).

Reply:
Yes. But rules_license currently uses the name license_text to mean the file.
So, there is going to be several rounds of iteration to get this right.

Reply:
Sounds good 👍